.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,149 @@
+---
+name: 🐛  Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in Cookiecutter!**
```

Just so that people don't confuse it with https://github.com/cookiecutter/cookiecutter
```suggestion
      **Thank you for wanting to report a bug in Open Science Lab's cookiecutter!**
```